### PR TITLE
Add the dropdown menu for the mobile site

### PR DESCRIPTION
**Pull request by Kanyorok**
I added the menu for the mobile version of the portfolio site. The idea is that the drop-down menu covers the whole page.

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @Kanyorok, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have JavaScript linter set up. Since you are using JavaScript in this project, kindly make sure you set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Kanyorok** [comment]
@LionRouge1 Hello, I have received your feedback. But, I did not understand. I have followed the steps for setting JavaScript, HTML, and CSS linter. You can see the file ".eslintrc.json" in the branch. Please, clarify what exactly should be changed/added.

File: js/menu.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function mobileMenu() {
```
Comment (BregornOriginal) [inline js/menu.js:1]: Remember that when a user clicks  (or taps) on any options of the mobile menu, the mobile menu should disappear

- Kindly check this functionality to make your requirements fulfilled 🥇

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Kanyorok and @khanh-devos  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good Pull Request description
✅ Good work following the design from Figma 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi Team @Kanyorok @khanh-devos ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No Linter Errors :100: 
- Meaningful Commit messages :+1: 

## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**